benchmark/buffers/buffer-creation.js

```diff
@@ -19,6 +19,7 @@ function main(conf) {
   const len = +conf.len;
   const n = +conf.n;
   switch (conf.type) {
+    case '':
```

Comment:
It would be good to add a short description why this is necessary.
E.g. `// Used for testing`

Reply:
Oy... I just landed this then saw this comment, my apologies for missing it